.circleci/config.yml

```diff
@@ -44,27 +44,11 @@ workflows:
             # Trigger the job also on git tag.
           tags:
             only: /^v.*/
-
           branches:
             ignore:
             - main
             - master
-    - architect/push-to-app-catalog:
-        context: architect
-        executor: app-build-suite
-        name: push-to-control-plane-app-catalog
-        app_catalog: control-plane-catalog
```

Comment:
is control-plane-catalog deprecated together with vintage?

Reply:
Yes because it's a cluster app in capi